### PR TITLE
formally deprecate config.json

### PR DESCRIPTION
**Pull request by lamont-granquist**
Signed-off-by: Lamont Granquist <lamont@scriptkiddie.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,16 +39,26 @@ Berkshelf is intended to be used as a CLI tool.  It is not intended to be used a
...
+The old config.json berkshelf file has been deprecated.
 
-You are encouraged to keep project-specific configuration in the `$PWD/.berkshelf` directory. A default configuration file is generated for you, but you can update the values to suit your needs.
+A partial list of common berkshelf settings and their Chef Config settings for conversion:
+
+| Berkshelf Config | Chef Config                       |
+|:---------------- |:--------------------------------- |
+| timeout          | rest_timeout                      |
+| headers          | headers                           |
+| client_name      | client_name                       |
```
Comment (coderanger) [inline README.md]: This should be `node_name` unless it is being set in a credentials file.

File: README.md [hunk 2]
```diff
@@ -39,16 +39,26 @@ Berkshelf is intended to be used as a CLI tool.  It is not intended to be used a
...
+| client_key       | signing_key_filename              |
```
Comment (coderanger) [inline README.md]: I think this one is backwards?

**Comment by koertkuipers** [comment]
what about berkshelfs `--config` switch? will it go away? or can i point it to a chef `config.rb` file?
currently --config still exists but i am not sure what it actually does. there is no easy way to switch which is needed when you for example `berks install` from one chef server and `berks upload` to another, a pretty typical situation for an environment cookbook.

**Comment by coderanger** [comment]
@koertkuipers You would use Chef config profiles for that.

**Comment by koertkuipers** [comment]
@coderanger thanks for that i read up on config.rb and profiles

**Pull request closed without merging**